src/views/governance/views/proposal/components/SimulateProposal.tsx

```diff
@@ -0,0 +1,149 @@
+import { Trans, t } from '@lingui/macro'
+import { LoadingButton } from 'components/button'
+import {
+  Box,
+  BoxProps,
+  Container,
+  Flex,
+  Image,
+  Link,
+  Spinner,
+  Text,
+  useColorMode,
+} from 'theme-ui'
+import { ROUTES, TENDERLY_SHARING_URL } from 'utils/constants'
+import { UsePrepareContractWriteConfig } from 'wagmi'
+import IssuanceIcon from 'components/icons/IssuanceIcon'
+import useProposalSimulation from '../hooks/useProposalSimulation'
+import { MODES } from 'components/dark-mode-toggle'
+import { TenderlySimulation } from 'types'
+import ExternalArrowIcon from 'components/icons/ExternalArrowIcon'
+
+interface Props extends BoxProps {
+  tx: UsePrepareContractWriteConfig
+}
+
+const getButtonStyles = (sim: TenderlySimulation | null) => {
+  if (!sim) return {}
+
+  return {
+    color: `var(--theme-ui-colors-${
```

Comment:
no need to use css variables, you can use the color name directly

Reply:
This is actually required for it to work properly. I tried it without and it doesn't